app/models/miq_policy.rb

```diff
@@ -353,7 +353,7 @@ def first_and_last_event
   end
 
   def self.seed
-    all.each do |p|
+    where(:towhat => nil).or(where(:active => nil)).or(where(:mode => nil)).each do |p|
```

Comment:
Hmm, that last comment begs to ask if we can just perform 3 updates for this instead of using ruby at all.